kompose.spec

```diff
@@ -235,20 +234,29 @@ done
 sort -u -o devel.file-list devel.file-list
 %endif
 
+# Why does check need to use buildroot at all? 
 %check
 %if 0%{?with_check} && 0%{?with_unit_test} && 0%{?with_devel}
 %if ! 0%{?with_bundled}
 export GOPATH=%{buildroot}/%{gopath}:%{gopath}
 %else
-export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
+#export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
+#export GOPATH=%{buildroot}/%{gopath}:$(pwd):%{gopath}
+export GOPATH=$(pwd):%{gopath}
```

Comment:
Just wondering, why does `check` need to use `%{buildroot}`? This didn't work for me because we aren't including any of the files from the vendor directory and so running `go test` didn't work because it couldn't find some of the deps it was expecting.

I had to use `GOPATH=$(pwd):%{gopath}` because otherwise go would find the pkg we wanted to test in the buildroot and try to test against that, but the buildroot doesn't have the vendor directory so it would fail.